contracts/utils/TablelandDeployments.sol

```diff
@@ -60,8 +57,6 @@ library TablelandDeployments {
             return ITablelandTables(POLYGON);
         } else if (block.chainid == 5) {
             return ITablelandTables(GOERLI);
-        } else if (block.chainid == 69) {
```

Comment:
I'm less familiar with this contract code here @sanderpick. Does removing this (and thereby changing the compiled contract code) have any significant impact?

Reply:
nope, just means people won't be able to use the deployments lib to get the kovan address. nbd.